examples/getting_started/pt/nvflare_pt_getting_started.ipynb

```diff
@@ -377,7 +399,30 @@
     "    executor = ScriptRunner(\n",
     "        script=\"src/cifar10_fl.py\", script_args=\"\"  # f\"--batch_size 32 --data_path /tmp/data/site-{i}\"\n",
     "    )\n",
-    "    job.to(executor, f\"site-{i+1}\")"
+    "    job.to(id=\"event_to_fed\", obj=executor, target=f\"site-{i+1}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a56abcd6-4e97-4a60-8894-2760f8815a03",
+   "metadata": {},
+   "source": [
+    "#### 8. Add TB Event\n",
+    "Add tensorboard logging to clients"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a8a733e0-c0a9-4c36-b49d-16b20c2df7f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nvflare.app_common.widgets.convert_to_fed_event import ConvertToFedEvent\n",
+    "\n",
+    "for i in range(n_clients):\n",
```

Comment:
Do you think for getting started maybe it's simpler to use job.to_clients() for this and the ScriptRunner?

Otherwise changes look good if we want this example to support metrics streaming. I'll update the other framework versions of the notebooks/site code blocks after this PR is merged.

Reply:
I think in this script we are using job.to for all places, so it would be better to avoid introducing variants like job.to_clients. Let's keep the current way.